app/templates/events/view/sessions.hbs

```diff
@@ -0,0 +1,117 @@
+<div class="ui buttons">
+  <button class="ui button">All</button>
+  <button class="ui button">Pending</button>
+  <button class="ui button">Accepted</button>
+  <button class="ui button">Confirmed</button>
+  <button class="ui button">Rejected</button>
+</div>
+<button class="ui blue button">Create Session</button>
+<button class="ui blue button">Export as CSV</button>
```

Comment:
We want two options. We want export as csv and export as pdf. So the drop down will contain the two fromats